### ISSUE TITLE
After update, service is not restarted

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/alba-asdmanager#146]
After restart we cannot remove an asd as the wizard keeps loading while calculating disk safety

**Comment by khenderick** [comment]
Seems related to the fact that the "current version" is 0, while it should be >= 1. This causes the migration part to be skipped where the service is restarted

**Comment by khenderick** [comment]
Fixed by #150, packaged in openvstorage-sdm-1.6.5-rev.423.713f250

**Comment by JeffreyDevloo** [comment]
### Steps
- Updated cluster
- Verified if SDM was running

### Output
```
ASD Manager running processes
=============================

alba-asd-6PHNeWA9DLTQ2F1D65PIHTxeCosyXqjP      start/running
alba-asd-A8Sl8PGJrOpX9lQDHZHWR4QQnQk0CcYk      start/running
alba-asd-DQVK0FaJAqSw5wetDjfUaehd1jzabVLh      start/running
alba-asd-Dyfxb6N3C6GZe2Pz440lqiV1YlaKc4MM      start/running
alba-asd-EONrkcQcr3MCZym006ZRvEmyXHGbjv3a      start/running
alba-asd-F9v38gwFjin3ISjNK5JDgp3NHRLk2wLX      start/running
alba-asd-GbedoAD2TkD7Px0UoymJwBPfPdirRKxt      start/running
alba-asd-SO8NIF1hBoaNYKLMniorjXZ3kouNXE98      start/running
alba-asd-Sy7zZs3bnNH8DS861LGTGLfbVwJTzu3V      start/running
alba-asd-YmfPZi2rtVsuEBQTytrYXtJr4KVqOw6i      start/running
alba-asd-sBaUDp6K1QvrPg7OwmaBiZCpanNTbE3a      start/running
alba-asd-xP2b0Cf7b5ddzYs94eDtxuRp4LNNrhe3      start/running
alba-maintenance_mybackend01-PLtLQnfycL95YJi8  start/running
asd-manager                                    start/running
asd-watcher                                    start/running

```
#### Updated from
```
- openvstorage                       2.7.3-fargo.1-1                                          amd64        openvStorage
- openvstorage-backend               1.7.3-fargo.1-1                                          amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.3-fargo.1-1                                          amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.3-fargo.1-1                                          amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-fargo.1-1                                          amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.3-fargo.1-1                                          amd64        openvStorage core
- openvstorage-hc                    1.7.3-fargo.1-1                                          amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.3-fargo.1-1                                          amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.3-fargo.1-1                                          amd64        openvStorage Web Applications
```

### Setup
- Ubuntu 14.04 (updated from Fargo 2.7.3)

#### Packages
```
- openvstorage                       2.7.4-fargo.1-1                                          amd64        openvStorage
- openvstorage-backend               1.7.5-fargo.1-1                                          amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.5-fargo.1-1                                          amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.5-fargo.1-1                                          amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-fargo.1-1                                          amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-fargo.1-1                                          amd64        openvStorage core
- openvstorage-hc                    1.7.5-fargo.1-1                                          amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.5-fargo.1-1                                          amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-fargo.1-1                                          amd64        openvStorage Web Applications

```

### Test result
Test passed.
